packages/node-cli/src/commands/object/import/ImportObjectsAction.js

```diff
@@ -138,4 +156,48 @@ module.exports = class ImportObjectsAction extends BaseAction {
 			}
 		}
 	}
+
+	async _getAllScriptIdsForObjectType(params) {
+		const sdkParams = {};
+		sdkParams.type = params[ANSWERS_NAMES.OBJECT_TYPE];
+
+		return this._callListObjects(params, sdkParams);
+	}
+
+	async _getAllScriptIds(params) {
+		return this._callListObjects(params, {});
+	}
```

Comment:
We are probably missing to forward the scriptid (if present) into the sdkParams

Reply:
Updated to cover this case as well.